frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -178,48 +208,39 @@ describe('MapProperties View', () => {
         </BrowserRouter>
       </Provider>,
     );
+    await act(() => (promise as unknown) as Promise<void>);
```

Comment:
the typing above `(promise as unknown) as Promise<void>)` seems a bit weird

What's wrong with `promise as Promise<void>` ?

Reply:
Typescript is angry with this format.  So that is the fix that I found

Reply:
sure, I'm just flagging thing that seem odd. to be addressed later. The type `unknown` is quite new to typescript and I don't know the difference with `any` but it feels that `as any` would be better suited here to force the typing. In any case, this works so leave as-is.